### PR TITLE
Calibration3D Pipeline Memory Leak

### PR DESCRIPTION
**Pull request by gerth2**
Applied patches from Banks to increase visibility of CVMat allocation & de-allocation.

Revised Calibration3D pipeline to release the unneded mat prior to returning.

Changed a few instances of using `opencv.Mat` to photon's `CVMat` for better traceability.

### CODE REVIEW

**Comment by gerth2** [comment]
Tested on both my PC and a Pi3 - verbose `--debug` trace log shows mat count is not spiraling out of control.

**Review by Bankst — DISMISSED** [review]

File: photon-server/src/main/java/org/photonvision/vision/opencv/CVMat.java [hunk 1]
```diff
@@ -43,23 +44,34 @@ public void copyTo(Mat srcMat) {
...
+            logStackTrace();
         }
     }
 
     @Override
     public void release() {
+        int matNo = allMats.get(mat);
         allMats.remove(mat);
         mat.release();
+        logger.trace(() -> "CVMat" + matNo + " de-alloc - new count: " + allMats.size());
```
Comment (pietroglyph) [inline photon-server/src/main/java/org/photonvision/vision/opencv/CVMat.java:73]: Should this be wrapped in a check for `shouldPrint`?

Comment (gerth2) [reply photon-server/src/main/java/org/photonvision/vision/opencv/CVMat.java:73]: likely

Comment (Bankst) [reply photon-server/src/main/java/org/photonvision/vision/opencv/CVMat.java:73]: yes and no - it won't print unless the Logger has been set to TRACE

**Review by Bankst — APPROVED** [review]

File: photon-server/src/main/java/org/photonvision/vision/opencv/CVMat.java (hunk 1 above)
Comment (gerth2) [reply photon-server/src/main/java/org/photonvision/vision/opencv/CVMat.java]: Per conversation - fixed it up to use lambdas better for performance.

**Review by Bankst — APPROVED** [review]

**Pull request merged**